rules/firestore/authenticated.test.js

```diff
@@ -0,0 +1,46 @@
+const firebase = require("@firebase/testing");
+const fs = require("fs");
+
+const projectId = `project-${Date.now()}`;
+let app;
+
+// Firestore emulator must be running
+// firebase serve --only firestore
```

Comment:
In the context of CI, is it recommended to run firebase serve? 
By that I mean, is there a way to start firebase serve, run tests (and regardless of result), kill the process of firebase serve?
I'd be interested to see how this behaves on Circle CI

Reply:
this needs some further investigation, I was just looking at our CI config as well and it could be that keeping all the rules separate with their own NPM package creates unnecessary work on the server. As for the emulation, I'm doing some further investigation as it looks like it [might not be straight forward on Circle](https://github.com/firebase/quickstart-nodejs/issues/48)